homeassistant/components/websocket_api/__init__.py

```diff
@@ -63,8 +62,8 @@ def async_register_command(
     handlers[command] = (handler, schema)
 
 
-async def async_setup(hass, config):
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     """Initialize the websocket API."""
-    hass.http.register_view(http.WebsocketAPIView)
+    hass.http.register_view(http.WebsocketAPIView())
```

Comment:
It'll be instantiated in `register_view` anyway. I think it's better to do this here to have cleaner `register_view` interface.